### PR TITLE
Fixed issue No 119 & 120, added 2 more resources for CSS

### PR DESCRIPTION
**Pull request by logichub**
Fixed following issues:
- https://github.com/tightenco/onramp/issues/119 - Add "Command Line Power User" by Wes Bos
- https://github.com/tightenco/onramp/issues/120 - Add content from Frontend Masters' new course?

Added content from following courses:
- [Wes Bos - CSS Grid](https://cssgrid.io/)
- [Wes Bos - What The Flexbox?!](http://flexbox.io/)

### CODE REVIEW

File: database/seeds/learn.php [hunk 1]
```diff
@@ -77,6 +77,21 @@
                     'url' => 'https://mastery.games/p/flexbox-zombies',
                     'type' => 'game',
                 ],
+                [
+                    'name' => 'Frontend Masters Web Development Bootcamp',
+                    'url' => 'https://frontendmasters.com/bootcamp/',
+                    'type' => 'course',
+                ],
+                [
```
Comment (mattstauffer) [inline database/seeds/learn.php]: I think CSS Grid and Flexbox are both "bonus." Do we have the idea of bonus represented in these seeds yet?

Comment (mattstauffer) [reply database/seeds/learn.php]: OK, I just added the ability to mark them as bonus. Just add the "bonus" keys and set them to true.

File: database/seeds/learn.php [hunk 2]
```diff
@@ -77,6 +77,21 @@
...
+                    'name' => 'Wes Bos - CSS Grid',
+                    'url' => 'https://cssgrid.io/',
+                    'type' => 'course',
```
Comment (mattstauffer) [inline database/seeds/learn.php]:
```suggestion
                    'type' => 'course',
                    'bonus' => true,
```

File: database/seeds/learn.php [hunk 3]
```diff
@@ -77,6 +77,21 @@
...
+                ],
+                [
+                    'name' => 'Wes Bos - What The Flexbox?!',
+                    'url' => 'http://flexbox.io/',
+                    'type' => 'course',
```
Comment (mattstauffer) [inline database/seeds/learn.php]:
```suggestion
                    'type' => 'course',
                    'bonus' => true,
```

File: database/seeds/learn.php (hunk 1 above)
Comment (logichub) [reply database/seeds/learn.php]: It seems to be a nice addition.

**Comment by mattstauffer** [comment]
Thanks!!

**Pull request merged**